k8s/pkg/apis/longhorn/v1beta1/backup.go

```diff
@@ -6,6 +6,7 @@ type BackupState string
 
 const (
 	BackupStateNew        = BackupState("")
+	BackupStatePending    = BackupState("Pending")
```

Comment:
NIT: technically this wasn't a valid state for v1beta1 but since it's not an ordered enum, it seems fine to keep it here :)

Reply:
Since it'd be backported to v1.2.4, thus I intended to add the Pending state for v1beta1.

Reply:
1.2.5 instead